src/vtcomposite.cpp

```diff
@@ -799,8 +799,8 @@ struct LocalizeWorker : Napi::AsyncWorker
                         }
 
                         else if (
-                            utils::startswith(property_key, baton_data_->class_property) ||
-                            utils::startswith(property_key, baton_data_->hidden_prefix + baton_data_->class_property))
+                            (property_key == baton_data_->class_property) ||
```

Comment:
will we still have a problem for tilesets having a property with name `class` (or whatever was passed in for the `baton_data_->class_property`)?